apps/meteor/client/components/Page/PageHeader.tsx

```diff
@@ -17,6 +17,11 @@ const PageHeader: FC<PageHeaderProps> = ({ children = undefined, title, onClickB
 	const [border] = useContext(PageContext);
 	const { isMobile } = useLayout();
 
+	const [currentRouteName] = useCurrentRoute();
+	if (!currentRouteName) {
+		throw new Error('No current route name');
+	}
+
```

Comment:
I dont think this is useful, this is a "raw" component it doesn’t matter if there is route or not, I think you kept this part after remove the back button

Reply:
solved